### ISSUE TITLE
Engraving Bench uncraftable

### ISSUE THREAD

**Issue by Bertel401** [GTNewHorizons/NewHorizons#1097]
While playing around with Railcraft i found out the Engraving Bench is not craftable.
Is there any reason this block is unobtainable?

I've tested with version 1.3.0.2 and version 1.3.0.5.

**Comment by Dream-Master** [comment]
Yes is disabled. I never found a use for it. I can make a recipe.

**Comment by VermillionX** [comment]
It's used to make emblems which can be attached to steam and electric locomotives as well as railcraft's metal posts and several other thingies. It's mostly an aesthetic thing.

**Comment by Dream-Master** [comment]
![image](https://cloud.githubusercontent.com/assets/2970688/22790983/35b9628a-eee8-11e6-85dd-266645512317.png)
